src/workerd/io/worker.h

```diff
@@ -711,6 +711,8 @@ class Worker::Actor final: public kj::Refcounted {
   // Runs an alarm in this actor. This function will handle deduplicating multiple alarms. If this
   // isn't a duplicate alarm, func will be called to run the alarm.
 
+  bool hasWebSocketHandlers();
```

Comment:
Needs doc comment.

I notice the implementation checks if the actor has *all* of the handlers. But I could imagine some apps won't actually need to do anything in particular to handle close or error messages, so might only provide a message handler. Should the check perhaps be OR-based, or only check for message handler?